plugin/ranger.vim

```diff
@@ -44,45 +44,40 @@ if !exists('s:choice_file_path')
   let s:choice_file_path = '/tmp/chosenfile'
 endif
 
+function! s:EditFiles(cmd)
+  try
+    if filereadable(s:choice_file_path)
+      let cwd = getcwd()
+      for f in readfile(s:choice_file_path)
+        exec a:cmd . ' ' . (f[0:len(cwd)-1] ==# cwd ? f[len(cwd)+1:-1] : f)
+      endfor
+      call delete(s:choice_file_path)
+    endif
+  endtry
+endfunction
+
+function! s:GetRangerCmd(path)
+  return s:ranger_command . ' --choosefiles=' . s:choice_file_path .
+      \ (isdirectory(a:path) ? ' "' : ' --selectfile="') . expand(a:path) . '"'
```

Comment:
I don't know why but the path is not concatenated correctly and that breaks the default behaviour:
- if you open a file in vim
- then you open ranger in the same window
- it should select the file in Ranger and it does not (see the command RangerCurrentFile).